zuul.d/project-templates.yaml

```diff
@@ -57,6 +57,26 @@
         - bionic-stein
         - bionic-queens
         - xenial-mitaka
+- project-template:
+    name: charm-yoga-functional-jobs
```

Comment:
I know this is a little picky, but these aren't really 'yoga' jobs.  These are xena, wallaby, etc. jobs as well.

I wonder if 'yoga' jobs should just be 'jammy-yoga' and 'focal-'yoga', and then the other sets should just be 'their' jobs too?